pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -134,10 +136,15 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxLinuxConfig(pod *v1.Pod) (
 		},
 	}
 
-	sysctls, err := getSysctlsFromAnnotations(pod.Annotations)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get sysctls from annotations %v for pod %q: %v", pod.Annotations, format.Pod(pod), err)
+	sysctls := make(map[string]string)
+	if utilfeature.DefaultFeatureGate.Enabled(features.Sysctls) {
+		if pod.Spec.SecurityContext != nil && len(pod.Spec.SecurityContext.Sysctls) > 0 {
```

Comment:
nit: `len(pod.Spec.SecurityContext.Sysctls) > 0` is not needed

Reply:
Done